unit-tests/py/common.py

```diff
@@ -0,0 +1,117 @@
+import os, sys, subprocess, traceback
+import pyrealsense2 as rs
+
+n_assertions = 0
+n_failed_assertions = 0
+n_tests = 0
+n_failed_tests = 0
+test_failed = False
+
+# If this is the first time running this script we set the wanted enviroment, However it is impossible to change the current running 
+# environment so we rerun the script in a child process that inherits the enviroment we set
+def set_env_vars(env_vars):
+    if len(sys.argv) < 2:
+        for env_var, val in env_vars.items():
+            os.environ[env_var] = val
+        sys.argv.append("rerun")
+        cmd = ["py", "-3"] + sys.argv
+        p = subprocess.run( cmd, stderr=subprocess.PIPE, universal_newlines=True )
+        exit(p.returncode)
+
+# Returns the first device that was found, if no device is found the test is aborted
+def get_first_device():
+    c = rs.context()
+    if not c.devices.size():  # if no device is connected we don't run the AC test
+        print("No device found, aborting AC test")
+        exit(0)
+    return c.devices[0]
+
+# Functions for asserting test cases:
+# Receives an expression which is an assertion. If false the assertion failed
+def require(exp):  
+    global n_assertions, n_failed_assertions, test_failed
+    n_assertions += 1
+    if not exp:
+        n_failed_assertions += 1
+        test_failed = True
+        print("Require failed")
+        for line in traceback.format_stack():
+            print(line.strip())
+
+# Receives the resulted value and the expected value and asserts they are equal
+def require_equal(result, expected):
+    global n_assertions, n_failed_assertions, test_failed
+    n_assertions += 1
+    if not result == expected:
```

Comment:
Can we check that the type of `expected` is not a list?

Reply:
We can, but I don't see why. Could you explain?

Reply:
So that someone won't try to compare lists here and would use the other function

Reply:
ok, I'll add the check